### DISCUSSION TITLE
Exporting multi responses created for a single Http Route

### DISCUSSION THREAD

**Discussion by saraalways100** [mockoon/mockoon#1321] [Help] [answered]
When i try to use "Export current environment to Open API v3 (JSON)" , its not exporting all the responses (more than one response) created for a Http Router. It exports only the active one.

**Comment by 255kb** [comment]
OpenAPI support is only [partial](https://mockoon.com/docs/latest/openapi/openapi-specification-compatibility/). Our system is currently able to export multiple responses, but as they are [mapped from the status code ](https://spec.openapis.org/oas/latest.html#patterned-fields-0), you may lose some information. 
If you have ideas to improve the way we export to OpenAPI feel free to share. 

Also, please note that not all of Mockoon's features necessarily have an equivalent in the OpenAPI specification.

**Reply by saraalways100** [reply]
Thanks for the reply.What i am trying to do is , take a backup of my work.I can see options to export in open api format only.Can you suggest me if there is a way to take a backup and restore it when i needed.

**Reply by 255kb** [reply, marked as answer]
OK, exporting in OpenAPI is definitely not a backup. 

Backuping your Mockoon setup is easy. Each mock API is in a separated JSON file.
You can locate it using the context menu, and even move the data file. Like this you can, for example, put it in a git tracked folder, or in a Dropbox/OneDrive/GDrive folder: https://mockoon.com/docs/latest/mockoon-data-files/data-storage-location/

You can even copy the whole config to the clipboard to quickly save it in a GitHub gist or file or anywhere else: 
![image](https://github.com/mockoon/mockoon/assets/7489814/ba305b6c-b5c5-4bb6-a8a2-346eb1643c64)
